### PR TITLE
Polar charts

### PR DESCRIPTION
**Pull request by boygirl**
cc/ @chrisbolin I ended up very much re-writing all the docs. I not really close to being finished, but I would love feedback before I continue down this path. I wont pick it up again until Monday

### CODE REVIEW

File: src/screens/docs/components/victory-scatter/ecology.md [hunk 1]
```diff
@@ -1,267 +1,466 @@
...
+```js
+  animate={{
+    duration: 2000,
+    onLoad: { duration: 1000 },
+    onEnter: { duration: 500, before: () => ({y: 0}) }
+  )}
+```
+### `bubbleProperty`
+
+The `bubbleProperty` prop indicates which property of the data object should be used to scale data points in a bubble chart. If a `bubbleProperty` is given, `size` and `symbol` props will be ignored. (Bubble charts should render circles).
```
Comment (chrisbolin) [inline src/screens/docs/components/victory-scatter/ecology.md]: > (Bubble charts should render circles)

Does this mean bubble charts **will** render circles, or is the user responsible for making that happen?

Comment (chrisbolin) [inline src/screens/docs/components/victory-scatter/ecology.md]: also, if it's the case that using bubbleProperty essentially makes a scatter chart a bubble chart, we should note that. that might help people grok what's going on

**Comment by chrisbolin** [comment]
@boygirl `victory@0.21.0` breaks a few examples that extended Area, Bar and Curve. I fixed those in #363 so no need to fix them here.

**Comment by chrisbolin** [comment]
this branch can be rebased with the current master just fine. I just pulled master, switched to this branch, and rebased. no conflicts, everything checks out. I didn't want to push those changes, because I didn't want to interrupt your workflow, Lauren :)

**Comment by chrisbolin** [comment]
Also, this branch currently **breaks sidebar search**.

jun-24-2017 16-46-24 docsmissing

I don't have any problem using a monospaced font for props, but I think we should do it with styles, rather than with `backticks`. It will also make it much easier to maintain. (And we won't be beholden to the code style with the changed background color)

File: src/screens/gallery/examples/continuously-animating-line.example.js [hunk 2]
```diff
@@ -1,62 +0,0 @@
-/* NOTE
```
Comment (boygirl) [inline src/screens/gallery/examples/continuously-animating-line.example.js]: Yes, the gallery generally needs to be audited. The examples there were placeholders that Paula grabbed from my list of fiddles. They aren't really organized or carefully considered.

File: src/screens/docs/components/victory-area/ecology.md [hunk 3]
```diff
@@ -1,260 +1,410 @@
...
-    {month: "September", profit: 35000, loss: 2000},
-    {month: "October", profit: 44000, loss: 8000},
-    {month: "November", profit: 55000, loss: 5000}
-  ]}
-  x="month"
-  y={(datum) => datum.profit - datum.loss}
-/>
+<VictoryChart
+  theme={VictoryTheme.material}
+  style={{
```
Comment (boygirl) [inline src/screens/docs/components/victory-area/ecology.md]: I see what you mean. I think I just wanted the first image on a given docs page to be instantly recognizable as "oh, that's what I want". I think wrapped and unwrapped is a good idea though.

File: src/screens/docs/components/victory-area/ecology.md [hunk 4]
```diff
@@ -1,260 +1,410 @@
...
+</VictoryChart>
 ```
 
 ## Props
 
-### data
+### `animate`
 
-Specify data via the `data` prop. By default, Victory components expect data as an array of objects with `x` and `y` properties. Use the [`x` and `y`] data accessor props to define a custom data format. The `data` prop must be given as an array.
+`VictoryArea` uses the standard `animate` prop. [Read about it here](https://formidable.com/open-source/victory/docs/common-props#animate)
```
Comment (boygirl) [inline src/screens/docs/components/victory-area/ecology.md]: I think for most of these common props the example is a good enough place to start, and the full explanation is just necessary for more complicated uses. Maybe putting the example first would make that better?

Comment (chrisbolin) [inline src/screens/docs/components/victory-area/ecology.md]: so just have the example from the common prop in VictoryArea etc, and the docs in common props?

**Comment by boygirl** [comment]
fixes #379
fixes #360
fixes #341 
fixes #345 
fixes #282
fixes #255

**Comment by boygirl** [comment]
Gallery page:
screencapture-localhost-3000-gallery-1499298018766

**Comment by chrisbolin** [comment]
Looks good to me! I just added some small fixes (see the above 6 commits). When CI passes I will merge, release, redeploy to www.

**Comment by chrisbolin** [comment]
live :) https://formidable.com/open-source/victory/guides/polar-charts

**Pull request merged**